src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -95,13 +95,13 @@ public CommandResult execute(Model model) throws CommandException {
     private static Person createEditedPerson(Person personToEdit, EditPersonDescriptor editPersonDescriptor) {
         assert personToEdit != null;
 
-        Name updatedName = editPersonDescriptor.getName().orElse(personToEdit.getName());
-        Phone updatedPhone = editPersonDescriptor.getPhone().orElse(personToEdit.getPhone());
-        Email updatedEmail = editPersonDescriptor.getEmail().orElse(personToEdit.getEmail());
-        Address updatedAddress = editPersonDescriptor.getAddress().orElse(personToEdit.getAddress());
-        Set<Tag> updatedTags = editPersonDescriptor.getTags().orElse(personToEdit.getTags());
-
-        return new Person(updatedName, updatedPhone, updatedEmail, updatedAddress, updatedTags);
+        //        Name updatedName = editPersonDescriptor.getName().orElse(personToEdit.getName());
+        //        Phone updatedPhone = editPersonDescriptor.getPhone().orElse(personToEdit.getPhone());
+        //        Email updatedEmail = editPersonDescriptor.getEmail().orElse(personToEdit.getEmail());
+        //        Address updatedAddress = editPersonDescriptor.getAddress().orElse(personToEdit.getAddress());
+        //        Set<Tag> updatedTags = editPersonDescriptor.getTags().orElse(personToEdit.getTags());
+        //        return new Person(updatedName, updatedPhone, updatedEmail, updatedAddress, updatedTags);
+        return null;
```

Comment:
Nice that you made this change in anticipation for a future feature. Will help us down the line!